### ISSUE TITLE
Support 'decorations_theme_variant' option on macOS?

### ISSUE THREAD

**Issue by bluz71** [alacritty/alacritty#6558]
Hello Alacritty team,

I prefer my desktop environment to use a light desktop theme, but  I much prefer a dark terminal.

On Linux the `window: decorations_theme_variant` allows one to override the desktop light/dark setting, I set it to dark which looks excellent since the title bar will now be dark blending perfectly with the dark terminal itself.

On macOS Alacritty uses the overall desktop setting (light or dark) to render the window title bar.
Is is possible to override that with a `window: decorations_theme_variant` approach?
Or does Alacritty take no part in rendering the title bar?
Basically is there a technical reason why `window: decorations_theme_variant` only applies to Linux GTK/Wayland and not in addition to macOS and Windows?

iTerm2 does provide a GUI option that mimics the Alacritty `window: decorations_theme_variant` approach FYIW.

Not the biggest or most important issue I know.
If possible to do, this would be excellent.
If impossible to do, then close this issue.

Best regards.

**Comment by chrisduerr** [comment]
I don't think dark theme can be set in our windowing library right now. It seems to be at least partially tracked upstream in https://github.com/rust-windowing/winit/issues/1549, though that seems to focus on an event for the theme change.

**Comment by bluz71** [comment]
Thanks Chris. Nice research, so definitely an upstream issue indeed.

I will monitor that winit issue, if/when that is done then we can look at leveraging that here. May happen, may not.

A good plan.

Cheers.

**Comment by bluz71** [comment]
Thanks @kchibisov.

Just confirming, will the existing `decorations_theme_variant` option be the trigger between forced light and forced dark Alacritty? (Like it is for Linux). I assume so.

That is, can I do the following to force Alacritty dark whilst having Mac desktop set to light:

```yml
window:
  decorations_theme_variant: dark
```

Best regards.

**Comment by chrisduerr** [comment]
Yes.
